### PR TITLE
Add discussion of formalization difficulties in symmetry breaking

### PR DESCRIPTION
**Pull request by JamesGallicchio**
Requested by reviewer 1. Not sure if there is anything more notable to share at that point in the paper.

### CODE REVIEW

**Comment by ccodel** [comment]
Not too much a fan of the casual-sounding word "wrangle," but other than that, looks good to me!

File: ITP/symmetry-breaking.tex [hunk 1]
```diff
@@ -130,3 +130,15 @@
 \end{align*}
 This concludes the proof.
 \end{proof}
+
+We had quite some difficulty formalizing the symmetry-breaking argument.
+We spent many hours around a whiteboard,
```
Comment (digama0) [inline ITP/symmetry-breaking.tex]: This line seems a bit unprofessional, I'm not sure we really need it. It seems better to characterize the difficulty as coming from the problem, with lines like "it is a delicate argument" or "it was challenging to find a formalization-friendly version of the reduction" without going into more details in the process itself. (If you have more information on why it's difficult, that's useful to add, but saying "we spent a while at the whiteboard" doesn't add more information from the reader's perspective beyond "it was difficult".)

Comment (bsubercaseaux) [reply ITP/symmetry-breaking.tex]: Agree with Mario

**Comment by ccodel** [comment]
Here's a rewrite of those two paragraphs. What do we think?

**Pull request merged**